spec/loading_spec.rb

```diff
@@ -35,6 +45,12 @@
         rv = system("ruby -e #{Shellwords.shellescape(code)}")
         expect(rv).to be true
       end
+
+      it 'produces no output' do
+        out, status = Open3.capture2e('ruby', '-w', stdin_data: code)
+        raise("Test script failed with exit status #{status.exitstatus}:\n#{out}") unless status.exitstatus == 0
```

Comment:
Let's use the `Process::Status` api for the check

```suggestion
        raise("Test script failed with exit status #{status.exitstatus}:\n#{out}") unless status.success?
```